packages/containers/examples/config/i18n.js

```diff
@@ -8,9 +8,10 @@ i18n.init({
 	resources: {
 		en: {
 			[I18N_DOMAIN_CONTAINERS]: {
-				DELETE_RESOURCE_MESSAGE: 'Are you sure you want to remove the {{resourceLabel}}',
-				DELETE_RESOURCE_MESSAGE_female: 'Are you sure you want to remove the {{resourceLabel}}',
-				DELETE_RESOURCE_QUESTION_MARK: '?',
+				DELETE_RESOURCE_MESSAGE:
+					'Are you sure you want to remove the <1>{{resourceType}}</1> <3><0>{{label}}</0></3> ?',
```

Comment:
Are these autogenerated? I don't understand what `<3><0>` means (two tags)?
I was looking on https://react.i18next.com/components/trans-component but didn't find any examples with two

Reply:
Yeah their doc is awful.
- yes it's generated by i18next-scanner, and follows how Trans works
- see `<index>` as the index of the react element child.

`Are you sure you want to remove the <1>{{resourceType}}</1> <3><0>{{label}}</0></3> ?`
`0`: Are you sure you want to remove the
`1`: {{resourceType}}
`2`: ' ' (space)
`<3/>`: `<strong/>` element that has 1 child
`<3><0/></3>`: the first child of `<3/>` -->  {{label}}

Reply:
Aah I see, thanks for explanation.

The only thing I see here is that these _types_ of translations might need to be verified by a dev before a translator touches it. For example if they change something that means the `<3>` should become a `<2>`, it's not so easy to know about that. 

But I don't think there's much to do about that on our side.

Reply:
The extraction is automatic, so no worries about that.
Then, if something changes on the value, the translation team will have a `modified` status. And they need to know that the `<number>` should be the same.